docs/source/datasources.rst

```diff
@@ -1,25 +1,26 @@
+###########
 Datasources
-===========
-
+###########
 StreamAlert supports the following services as datasources:
 
 * Amazon S3
-* AWS Kinesis Streams
-* AWS SNS
+* Amazon Kinesis Data Streams
+* Amazon Simple Notification Service (SNS)
 
 These services above can accept data from:
 
 * Log Forwarding Agents
 * Custom Applications
 * AWS CloudTrail
```

Comment:
Also replace `AWS` with `Amazon` here, since you replace most of them. 🤷‍♀

Reply:
sadly, they (Amazon/AWS) use different terminology for different services, and I have no idea why... but, all the changes I made were to match what they refer to them as:

## `AWS CloudTrail`
![image](https://user-images.githubusercontent.com/13773789/74800566-b8f03000-5288-11ea-8cee-70527fa440e8.png)



## Versus `Amazon S3`
![image](https://user-images.githubusercontent.com/13773789/74800591-cad1d300-5288-11ea-86c2-31bb6f5cf2d4.png)